python/test/test_bug.py

```diff
@@ -1,6 +1,8 @@
 import zipfile
 from tempfile import NamedTemporaryFile
 
+from pandas.core.algorithms import isin
```

Comment:
what is `isin` used for?

Reply:
oops did not see that pull in. must of been some auto import on something. will take it out.